<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contact_edit_Form</class>
 <widget class="QWidget" name="Contact_edit_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Contact edit</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>510</height>
    </rect>
   </property>
   <widget class="QTextEdit" name="Name_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>389</width>
      <height>50</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextEditorInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">address:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Address_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>389</width>
      <height>100</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextEditorInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>250</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">phone numbers (separated by semicolons):</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Phone_numbers_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>389</width>
      <height>50</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextEditorInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>250</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">email (if many - separated by semicolons):</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Email_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>389</width>
      <height>50</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextEditorInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">comments:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Comments_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>389</width>
      <height>100</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextEditorInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="OK_pushButton">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
